mpas_analysis/__main__.py

```diff
@@ -205,6 +205,9 @@ def build_analysis_list(config, controlConfig):
     analyses.append(ocean.WoceTransects(config, oceanClimatolgyTasks['avg'],
                                         controlConfig))
 
+    analyses.append(ocean.OsnapTransects(config, oceanClimatolgyTasks['avg'],
+                                        controlConfig))
+    
```

Comment:
You'll need to rebase onto the current `develop` when you get a chance.  After doing that, it will be important to fix this typo that got fixed elsewhere in this file.
```suggestion
    analyses.append(ocean.OsnapTransects(config, oceanClimatologyTasks['avg'],
                                        controlConfig))
    
```